data/settings.py

```
@@ -6,12 +6,14 @@
 
 import dj_database_url
 import rediscluster
+from dotenv import find_dotenv, load_dotenv
```

will this be true in production? env vars will come from PaaS in that case. is this file only loaded in development mode?

---

dotenv creates environment variables from a .env file if it exists if not it just fails silently. This fragment of code will be executed in all environments, but of course only once, when the app starts, and it is harmful if there is no .env file present which I assume is the case in production/staging. Am I correct?

---

You are, in which case this is fine